lib/datadog/appsec/configuration/settings.rb

```diff
@@ -95,6 +95,62 @@ def self.add_settings!(base)
                 end
               end
 
+              settings :block do
+                # HTTP status code to block with
+                option :status do |o|
+                  o.default { 403 }
+                end
+
+                # only applies to redirect status codes
+                option :location do |o|
+                  o.setter { |v| URI(v) unless v.nil? }
```

Comment:
Steep insists that the `v` in `URI(v)` can be `nil` here

Reply:
I mean, `URI(v) unless v.nil?` should make it so that `v` in `URI(v)` is not, but Steep still believes it can.